src/home_robot_hw/home_robot_hw/nodes/goto_controller.py

```diff
@@ -85,17 +85,18 @@ def _goal_update_callback(self, msg: Pose):
         self.goal_visualizer(pose_goal.matrix())
 
     def _enable_service(self, request):
+        self.xyt_goal = None
         self.active = True
         return TriggerResponse(
             success=True,
-            message=f"Goto controller is now RUNNING",
+            message="Goto controller is now RUNNING",
```

Comment:
why this change?

Reply:
Incorrect usage of f-strings (without a placeholder)